src/webserver/SslWsSession.h

```diff
@@ -198,18 +127,9 @@ class SslWsUpgrader : public std::enable_shared_from_this<SslWsUpgrader>
         // The websocket::stream uses its own timeout settings.
         boost::beast::get_lowest_layer(https_).expires_never();
 
-        std::make_shared<SslWsSession>(
-            ioc_,
-            std::move(https_),
-            ip_,
-            backend_,
-            rpcEngine_,
-            subscriptions_,
-            balancer_,
-            etl_,
-            tagFactory_,
-            dosGuard_,
-            std::move(buffer_))
+        std::make_shared<SslWsSession<Handler>>(
+            std::move(https_), ip_, this->tagFactory_, this->dosGuard_, this->handler_, std::move(buffer_))
```

Comment:
Again i wonder why `this->` is used on some but not others?

Reply:
It is because the base class is a template class. More info see :https://stackoverflow.com/questions/4643074/why-do-i-have-to-access-template-base-class-members-through-the-this-pointer